assets/i18n/hi.json

```diff
@@ -38,6 +38,7 @@
   "I18N_ERROR_PAGE_TITLE_500": "Error 500 - Oppia",
   "I18N_FOOTER_ABOUT": "हमारे बारे में",
   "I18N_FOOTER_ABOUT_ALL_CAPS": "OPPIA के बारे में",
+  "I18N_FOOTER_AUTHOR_PROFILE": "",
```

Comment:
@joshuacano, I got some translations (thanks to @raju249!) Here they are:

AUTHOR PROFILES: लेखक प्रोफाइल
Share this exploration: शेयर दीस एक्सप्लोरेशन